Sources/Conformance/main.swift

```diff
@@ -120,6 +120,8 @@ func singleTest() throws -> Bool {
 }
 
 Google_Protobuf_Any.register(messageType: Conformance_TestAllTypes.self)
+Google_Protobuf_Any.register(messageType: ProtobufTestMessages_Proto3_TestAllTypes.self)
```

Comment:
Also, to be clear:  The registry is *only* needed if you are transcoding messages between different encodings (e.g., protobuf-to-JSON or vice-versa).  It's not needed in any other case.  (In particular, I do not need any registry to decode JSON and recode back to JSON, unlike the C++ implementation which relies on the descriptor registry in this case.)